### ISSUE TITLE
Need more practical examples for real applications

### ISSUE THREAD

**Issue by yptheangel** [yptheangel/opencv-starter-pack#2]
Looking for PRs that include good starter code or good sample applications for OpenCV app devs

**Comment by sainimohit23** [comment]
Hi @yptheangel, can I work on this?

**Comment by yptheangel** [comment]
@sainimohit23 yes! you are most welcome. Do you have any ideas on what starter code to add?

I have a few interesting ones, but please feel free to give your suggestions
### todo
**deep learning based**
1. human detection using opencv dnn module (dont use pytorch/tf/keras)
2. ocr using tesseract
3. license plate recognition using tesseract

**classical cv but useful**
1. warping using cv2.warpPerspective
2. thresholding
